nexus/db-queries/src/db/datastore/v2p_mapping.rs

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use super::DataStore;
+use crate::context::OpContext;
+use crate::db;
+use crate::db::error::{public_error_from_diesel, ErrorHandler};
+use crate::db::model::V2PMappingView;
+use async_bb8_diesel::AsyncRunQueryDsl;
+use diesel::{QueryDsl, SelectableHelper};
+use omicron_common::api::external::ListResultVec;
+
+impl DataStore {
+    pub async fn v2p_mappings(
```

Comment:
Well... I admittedly tend to use `DataPageParams::max_page()` in RPWs whenever using db queries that require pagination parameters, and that will return `u32::MAX` (`4_294_967_296`) entries, so I'd say it's kind of moot here, unless we want to standardize on a lower limit for db queries in background tasks (which honestly, it wouldn't shock me if we decide that it would be a good idea to do so!)

Reply:
I have updated the query to use this pattern.